src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/FormMetadataProvider.php

```diff
@@ -28,12 +28,29 @@ class FormMetadataProvider implements MetadataProviderInterface
      */
     private $formMetadataLoaders;
 
+    /**
+     * @var string
+     */
+    private $defaultLocale;
+
+    /**
+     * @param string[]|null $locales
+     */
     public function __construct(
         ExpressionLanguage $expressionLanguage,
-        iterable $formMetadataLoaders
+        iterable $formMetadataLoaders,
+        ?array $locales = []
```

Comment:
i think we need to add this parameter to the service definition? 🤔

Reply:
i guess you are injecting the configured locales here - maybe we can call it `systemLocales`?

Reply:
or maybe directly use the `sulu_core.fallback_locale`?

Reply:
Oh did search for default_locale but not for fallback_locale, will directly use here the fallback locale instead of sulu_core.locales.